.ci_helpers/run-test.py

```diff
@@ -0,0 +1,50 @@
+"""run-test.py
+
+Script to run tests in Github.
+
+"""
+import argparse
+import glob
+import os
+from pathlib import Path
+
+import pytest
+
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser(description="Run tests listed.")
+    parser.add_argument("touchedfiles", metavar="TOUCHED_FILES", type=str)
+    args = parser.parse_args()
+    file_list = args.touchedfiles.split(",")
+
+    pytest_args = ["--log-cli-level=WARNING", "-vv"]
+    test_to_run = {}
+    for f in file_list:
+        file_path = Path(f)
+        file_name, file_ext = os.path.splitext(os.path.basename(f))
+        if file_ext == ".py":
+            if any(
+                [
+                    (file_path.match("echopype/convert/*")),
+                    (file_path.match("echopype/tests/test_convert*")),
+                ]
+            ):
+                if "convert" not in test_to_run:
+                    test_to_run["convert"] = []
+                test_to_run["convert"].append(file_path)
+            elif any(
+                [
+                    (file_path.match("echopype/calibrate/*")),
+                    (file_path.match("echopype/tests/test_calibrate*")),
+                ]
+            ):
+                if "calibrate" not in test_to_run:
+                    test_to_run["calibrate"] = []
+                test_to_run["calibrate"].append(file_path)
```

Comment:
@leewujung I think this captured your desired capability ... Pytest now runs the appropriate tests based on the folder path of touched files. I included 2 patterns here:
1. Anything under `echopype/convert/`
2. Modified or added tests files that has pattern `test_convert`

I am currently only capturing calibrate and convert, but more can be added in the future.

Reply:
Awesome! I guess we'll see it at work when I make the changes to open_raw we discussed yesterday. Excited!